ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -804,6 +804,59 @@ public func timer(interval: NSTimeInterval, onScheduler scheduler: DateScheduler
 	}
 }
 
+/// Creates a repeating timer of the variable interval starting with `nextInterval(0)`,
+/// with a reasonable default leeway, sending updates on the given scheduler.
```

Comment:
> But I'm wondering if we should accept fixed leeway for variable timer intervals. Maybe replacing to leewayFactor (default = 0.1) will be better.

That might be a good idea (for `timer` as well). 
I'm wondering if thats also the reason that there are two `timer` functions, one with a default leeway and one with leeway as a parameter instead of using a default value for the second one and saving the variant with default leeway? 
By changing it to `leewayFactor`, it could be changed to 

```public func timer(interval: NSTimeInterval, onScheduler scheduler: DateSchedulerType, withLeeway leewayFactor: Float = 0.1) -> SignalProducer<NSDate, NoError> {```

and same for `variableTimer` I think.